src/components/styles/ChooseType/ProjectSlide.css

```diff
@@ -98,7 +98,7 @@
   padding-left: 12.5%;
   padding-right: 12.5%;
   border-radius: 3px;
-  background-color: rgb(199, 194, 212);
+  background-color: rgb(164, 162, 179);
```

Comment:
Global variable change needed

Reply:
I deleted (actually made it into comment or sth) all the description:hover.